### ISSUE TITLE
[BUG]: Fallo CAS Login / Guest 

### ISSUE THREAD

**Issue by Acr0l** [open-source-uc/planner#273]
**Descripción**
Siguiendo el README en GH Codespaces y utilizando `just init` no se logra un funcionamiento óptimo del planner, ya que toda acción te manda un 5XX (Internal Server Error)

**Pasos para generar el bug**
Pasos para replicar el comportamiento:
1. Open in Codespaces
2. Terminal `just init`
3. Run & Debug (Launch All)

**Output de terminal**
Exception has occurred: ModuleNotFoundError

No module named 'prisma.models'
  File "/workspaces/planner/backend/app/routes/admin.py", line 2, in <module>
    from prisma.models import (
  File "/workspaces/planner/backend/app/routes/__init__.py", line 5, in <module>
    from . import admin, course, offer, plan, user
  File "/workspaces/planner/backend/app/main.py", line 5, in <module>
    from . import routes, sync
ModuleNotFoundError: No module named 'prisma.models'

(CAS Server Failure)

npx --yes cas-server-mock --database=$PWD/cas-mock/data/cas-mock-users.json
Debugger attached.
npm WARN deprecated consolidate@0.16.0: Please upgrade to consolidate v1.0.0+ as it has been modernized with several long-awaited fixes implemented. Maintenance is supported by Forward Email at https://forwardemail.net ; follow/watch https://github.com/ladjs/consolidate for updates and release changelog
npm WARN deprecated angular-route@1.6.6: For the actively supported Angular, see https://www.npmjs.com/package/@angular/core. AngularJS support has officially ended. For extended AngularJS support options, see https://goo.gle/angularjs-path-forward.
npm WARN deprecated angular@1.6.6: For the actively supported Angular, see https://www.npmjs.com/package/@angular/core. AngularJS support has officially ended. For extended AngularJS support options, see https://goo.gle/angularjs-path-forward.
Debugger attached.
Waiting for the debugger to disconnect...
/home/vscode/.npm/_npx/72d83113b006f95c/node_modules/cas-server-mock/server.js:27
  throw new Error(`Invalid arguments: ${error.message}`);
  ^

Error: Invalid arguments: Cannot find module '/workspaces/planner/cas-mock/data/cas-mock-users.json'
Require stack:
- /home/vscode/.npm/_npx/72d83113b006f95c/node_modules/cas-server-mock/server.js
    at Object.<anonymous> (/home/vscode/.npm/_npx/72d83113b006f95c/node_modules/cas-server-mock/server.js:27:9)
    at Module._compile (node:internal/modules/cjs/loader:1198:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1252:10)
    at Module.load (node:internal/modules/cjs/loader:1076:32)
    at Function.Module._load (node:internal/modules/cjs/loader:911:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:22:47
Waiting for the debugger to disconnect...

**Comportamiento esperado**
Acceso al planner. Pero en CAS Login se va a negro.

**Screenshots**
![image](https://github.com/open-source-uc/planner/assets/66649568/890a51a7-d09b-41e3-bbc0-5fb0ebdb0c77)
  

</details>

**Escritorio (Información requerida):**
 - GH Codespaces

**Comment by agucova** [comment]
El primer error es consistente con la ausencia del cliente de Prisma, lo que es raro, porque el `db-reset` en el init debiese generarlo automáticamente.

El segundo error es cosnsitente con un `cas-mock-users` ausente, lo que también debiese generarse con `just init`.

Es posible que esto haya sido un error temporal en main.

Podrías pullear y correr `just init` de nuevo y ver si funciona? También me interesa ver el output de `just init`.

**Comment by Acr0l** [comment]
Sí, fue un problema iniciali de `just init` que no había sido ejecutado correctamente. Pero al aplicar los cambios tengo unos problemas con el redirect. Creo que es porque estoy trabajando desde un Codespace que hace que algunos links no funcionen.

**Comment by benjavicente** [comment]
¿Donde estás corriendo el Codespace? Puede que en el navegador no funcione todo correctamente, si no lo has hecho prueba corriendo el Codespace en VSCode

**Comment by agucova** [comment]
@benjavicente creo que es el mismo error que tuviste tú. En principio ya soportamos arreglarlo (ahora tenemos las 2 env variables), pero ese no es el .env por defecto.

Podríamos configurar el `just init` para detectar codespaces y si es el caso mover un .env distinto

**Comment by agucova** [comment]
@Acr0l como dijo Benja, la solución por ahora es correr el devcontainer en tu computador en vez de Codespaces.

**Comment by Acr0l** [comment]
Lo desarrollé local y ahora tengo un problema con el backend... quizá no hice bien el Dev Container

![image](https://github.com/open-source-uc/planner/assets/66649568/733a74a0-6679-437b-b9f2-8f64c2c0dd44)
(Problema al R&D)

**Comment by agucova** [comment]
@Acr0l Estamos en Discord haciendo un sprint de planner, asi que si quieres te puedes sumar y te ayudamos

Creo que te escribí a tu telegram, pero no si todavia lo tienes activo, tmb me puedes hablar a IG @agustin_covarrubias y te paso el link jajaja
